src/components/editor/previews/ConditionalPreview.tsx

```diff
@@ -36,6 +36,34 @@ const ConditionalPreview: React.FC<{ component: IComponent }> = ({
       )}
     </Box>
   )
+} */
+
+const ConditionalPreview: React.FC<IPreviewProps> = ({ component }) => {
+  const acceptedTypes = [
+    'Box',
+  ] as ComponentType[]
+  const { props, ref } = useInteractive(component, true)
+  const { drop, isOver } = useDropComponent(component.id, acceptedTypes)
+
+
+  if (isOver) {
+    props.bg = 'teal.50'
+  }
+
+  return (
+    <Box pos="relative" {...props} ref={drop(ref)}>
+      {props.show === 'show-both' || props.show === undefined ? (
+        <>
+          <ComponentPreview componentName={component.children[0]} />
+          <ComponentPreview componentName={component.children[1]} />
```

Comment:
what if I delete any box

Reply:
the box gets deleted as expected, and only one box is displayed, if both are deleted then nothing is displayed, no error is thrown, but I'll add the null check

Reply:
Also I tried removing the box under conditional, but it ceases to become a meta component when I do that, and the conditional component ends up having no box in it by default

Reply:
Okay